entgql/schema.go

```diff
@@ -324,6 +332,19 @@ func (e *schemaGenerator) typeFromField(f *gen.Field, idField bool, userDefinedT
 		scalar = scalar[strings.LastIndexByte(scalar, '.')+1:]
 		return namedType(scalar, nillable), nil
 	case typ == field.TypeJSON:
+		if f.Type.RType != nil {
+			switch f.Type.RType.Kind {
+			case reflect.Slice, reflect.Array:
+				switch f.Type.RType.Ident {
+				case "[]float64":
+					return listNamedType("Float", f.Optional), nil
+				case "[]int":
+					return listNamedType("Int", f.Optional), nil
+				case "[]string":
+					return listNamedType("String", f.Optional), nil
```

Comment:
This is generated as `[String!]`, but the `Optional` attribute is on the slice, not its elements. i.e. `[String]!`

Reply:
We can't put a nil value in the string slide, right?